src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPGlobalProperties.cs

```diff
@@ -61,7 +61,6 @@ internal static IPGlobalProperties InternalGetIPGlobalProperties()
 
         public abstract IPGlobalStatistics GetIPv6GlobalStatistics();
 
-        /// Returns a list of all unicast IP addresses after ensuring they are all stable.
```

Comment:
These all need to be in <summary> tags if we want them to enable Intellisense. I'll clean up this file and put them all in proper tags for Intellisense.